src/ResourceManager/DevSpaces/Commands.DevSpaces.Test/ScenarioTests/DevSpacesTests.ps1

```diff
@@ -0,0 +1,80 @@
+<#
+.SYNOPSIS
+Test DevSpaces stuff
+#>
+function Test-AzureRmDevSpacesController
+{
+    # Manual Setup. Create these resource maunally in the subscription
+    $resourceGroupName = "rgps4504"
+    $kubeClusterName = "kubeps5496"
+
+    # Setup
+    $devSpacesName = Get-DevSpacesControllerName
+    $tagKey =  Get-DevSpacesControllerTagKey
+    $tagValue =  Get-DevSpacesControllerTagValue
+
+    $referenceObject = @{}
+    $referenceObject.Name = $devSpacesName
+    $referenceObject.ResourceGroupName = $resourceGroupName
+    $referenceObject.ProvisioningState = "Succeeded"
+    $referenceObject.Location = "eastus"
+
+    #Create new dev spaces
+    $devSpaceController = New-AzureRmDevSpacesController -ResourceGroupName $resourceGroupName -Name $devSpacesName -TargetClusterName $kubeClusterName -TargetResourceGroupName $resourceGroupName
+    Assert-AreEqualPSController $referenceObject $devSpaceController
+
+    #Get dev spaces
+    $devSpaceController = Get-AzureRmDevSpacesController -ResourceGroupName $resourceGroupName -Name $devSpacesName
+    Assert-AreEqualPSController $referenceObject $devSpaceController
+
+    #update dev spaces
+    $devSpaceControllerUpdated = $devSpaceController | Update-AzureRmDevSpacesController -Tag @{ $tagKey=$tagValue}
+    Assert-AreEqualPSController $referenceObject $devSpaceControllerUpdated
+    $tag = Get-AzureRmTag -Name $tagKey
```

Comment:
This seems like overkill - can you just say 
```
Assert-AreEqual $devSpaceControllerUpdated.Tag.Values.Name $tagName
Assert-AreEqual $devSpaceControllerUpdated.Tag[tagName] $tagValue
```

Reply:
Update-AzureRMDevSpaces controller does not return tags.